### DISCUSSION TITLE
Left side bar color

### DISCUSSION THREAD

**Discussion by mmip-labs** [netbox-community/netbox#17440] [Help Wanted!]
Hi, there! 
I have updated the Netbox to 4.1.0 In this version, the color of the left side bar has been changed and is not comfortable for me. How can i change it? What CSS should i change?

**Comment by ghost** [comment]
you can't without modifying code and having a local fork.
NBL decided for you.

**Reply by mmip-labs** [reply]
What is NBL?)

**Reply by ghost** [reply]
netboxlabs

**Comment by jeremystretch** [comment]
Can't make [everyone happy](https://github.com/netbox-community/netbox/issues/15615#issuecomment-2071868760).

**Reply by ghost** [reply]
Let's be real though, it's because you didn't like the idea. (Ref: https://github.com/netbox-community/netbox/issues/15615, https://github.com/netbox-community/netbox/issues/16393, https://github.com/netbox-community/netbox/issues/16876)
In https://github.com/netbox-community/netbox/issues/16876 you specifically say people are invited to share their thoughts on the styling, then when people do (like OP) you basically say ¯\\\_(ツ)\_/¯

**Reply by jeremystretch** [reply]
The sidebar color was changed like Brian asked. @mmip-labs complained that it was. I agree with him that the original style was preferable.
